Detect and fix platform registration failures (#132)
canonical/tdx: README.md

	The platform registration is done with the `mpa_registration_tool` tool.
	This service is executed on system start up, registers the platform, and gets deactivated.
	Please check the following two logs to confirm successful registration:

	1. Check service logs with following command:

		```bash
		sudo systemctl status mpa_registration_tool
		```

> 		An example output:

Reviewer: ```suggestion
		A successful example output:
```

		```bash
		mpa_registration_tool.service - Intel MPA Registration
			Loaded: loaded (/usr/lib/systemd/system/mpa_registration_tool.service; enabled; preset: enabled)
			Active: inactive (dead) since Tue 2024-04-09 22:54:50 UTC; 11h ago
		Duration: 46ms
		Main PID: 3409 (code=exited, status=0/SUCCESS)
				CPU: 21ms

		Apr 09 22:54:50 right-glider-515046 systemd[1]: Started mpa_registration_tool.service - Intel MPA Registratio>
		Apr 09 22:54:50 right-glider-515046 systemd[1]: mpa_registration_tool.service: Deactivated successfully.
		```

> 	2. Check plateform registration logs with following command:

Reviewer: ```suggestion
	2. Check MPA log file with following command:
```

		```bash
		cat /var/log/mpa_registration.log
		```

> 		An example output:

Reviewer: ```suggestion
		A successful example output:
```

> 	Apr 09 22:54:50 right-glider-515046 systemd[1]: Started mpa_registration_tool.service - Intel MPA Registratio>

Reviewer: Please change the description further above to better integrate your additions.
Change "Please check the service does not output any error message:" to "Please check the following two logs to confirm successful registration:".
Then, add a number list and the first point is: "Check service log with following command:"

	Apr 09 22:54:50 right-glider-515046 systemd[1]: mpa_registration_tool.service: Deactivated successfully.
	```

> 	Also file `/var/log/mpa_registration.log` is expected to have below logs for
> 	successful `Platform registration`:

Reviewer: ```suggestion

	2. Check log file `/var/log/mpa_registration.log`.
	A successful example output:
```

	```bash
	[04-06-2024 03:05:53] INFO: SGX Registration Agent version: 1.20.100.2
	[04-06-2024 03:05:53] INFO: Starts Registration Agent Flow.
	[04-06-2024 03:05:54] INFO: Registration Flow - PLATFORM_ESTABLISHMENT or TCB_RECOVERY passed successfully.
	[04-06-2024 03:05:54] INFO: Finished Registration Agent Flow.
    ```

> 	If the failure occurred, you might want to boot into the BIOS and Go to 
> 	`Socket Configuration > Processor Configuration > Software Guard Extension (SGX)` and Set

Reviewer: ```suggestion
	If an error is reported in one of the logs, boot into the BIOS, go to 
	`Socket Configuration > Processor Configuration > Software Guard Extension (SGX)`, and set
```

> 	- `SGX factory reset` to `Enabled`

Reviewer: ```suggestion
	- `SGX Factory Reset` to `Enabled`
```
